managed/services/supervisord/supervisord.go

```diff
@@ -419,23 +428,40 @@ func (s *Service) marshalConfig(tmpl *template.Template, settings *models.Settin
 	clickhousePoolSize := getValueFromENV("PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE", "")
 	clickhouseBlockSize := getValueFromENV("PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE", "")
 	clickhouseAddrPair := strings.SplitN(clickhouseAddr, ":", 2)
+	vmSearchDisableCache := getValueFromENV("VM_search_disableCache", strconv.FormatBool(!settings.VictoriaMetrics.CacheEnabled))
```

Comment:
Any reason why env vars are not uppercase in case of VM_ prefix?

Reply:
Yes, the naming is opinionated and comes from VM command line parameters. They have adapted those to be envvars as well, hence the weird naming.

Reply:
You can check the docs:
- https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#list-of-command-line-flags
- https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#environment-variables